flexiblepricing/utils.py

```diff
@@ -0,0 +1,80 @@
+from cms.models import FlexiblePricingRequestForm, FormField
+from django.db.models import Q
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def ensure_flexprice_form_fields(form_instance: FlexiblePricingRequestForm):
+    """
+    Checks the supplied form instance for the proper Flexible Pricing Request Form
+    fields and adds them if it needs to, or logs if it gets confused. This is
+    meant to be called by the signal receiver (see cms/signalreceivers.py) or by
+    a management command.
+
+    Args:
+    - form_instance (FlexiblePricingForm) - the form to work on
+
+    Returns:
+    - boolean - True if the form was OK, False otherwise
+    """
+
+    currency_field = False
+    income_field = False
+
+    relevant_fields = form_instance.form_fields.filter(
+        Q(field_type__in=["country", "number"])
+        | Q(clean_name__in=["your_income", "income_currency"])
+    ).all()
+
+    if len(relevant_fields) != 2:
+        logger.warning(
+            f"Improper length {len(relevant_fields)} returned; was expecting 2"
+        )
+
+    for field in relevant_fields:
+        if field.field_type == "country":
+            if field.clean_name == "income_currency":
```

Comment:
No need to change, but for me, it looks clearer if two if are combined in one line.